public/locales/en/common.json

```diff
@@ -261,12 +265,12 @@
       "title": "Set bounty as draft"
     },
     "mobile-information": {
-      "our-web-application": "Our web application currently does not support a mobile version.",
-      "if-you-want": "If you want to use it you must connect to our web application via desktop.",
-      "stay-updated": "Stay updated on"
+      "title": "MetaMask Mobile",
+      "description": "MetaMask is not available on mobile browsers. In order to interact with this DApp on mobile you need to access using Metamask app built in browser.",
```

Comment:
I would rewrite to "MetaMask is not available _on this mobile browser_. In order to interact with this DApp on a mobile you will to access using MetaMask built-in browser"